README.md

```diff
@@ -799,3 +809,8 @@ flowchart TD
     click F "https://github.com/dodevops/cloudcontrol/blob/develop/.github/workflows/test.yml" "Test workflow"
     click H "https://github.com/dodevops/cloudcontrol/blob/develop/.github/workflows/release.yml" "Release workflow"
 ```mermaid
+maid
```

Comment:
You should check your generator/template code for the mermaid stuff - looks sus

Reply:
Hm. Running it again fixes it. 🤷🏽